web/src/features/map/MapTooltip.stories.tsx

```diff
@@ -8,7 +8,7 @@ interface StoryStateZoneData {
   ci: number;
   fr: number;
   rr: number;
-  e?: boolean | null;
+  e?: string | number | null;
```

Comment:
To adress your other comment we could switch this to be the `estimated ratio/percentage` and add a new one for the `estimation method`.

That should simplify things in the code itself and make it more flexible for the future.